doc/tutorial.rst

```diff
@@ -586,8 +573,9 @@ The `static/js/src/simplevideo_edit.js` JavaScript file should look like this: :
           maxwidth: $(element).find('input[name=maxwidth]').val(),
           maxheight: $(element).find('input[name=maxheight]').val()
         };
+        runtime.notify('save', {state: 'start'});
         $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
-          window.location.reload(false);
+          runtime.notify('save', {state: 'end'});
```

Comment:
@mtyaka Yes, I had to remove the vertical tag from xblock-poll to make it work. It does beg the question of why it's there to begin with. Perhaps we should consult with upstream about removing the vertical tag from the default as well. Do we know why it's included to begin with?